derFabio/Übung2.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Webshop Animal Farm</title>
+</head>
+<body>
+    <header>
+        <h1>Welcome to the Animal Farm Webshop</h1>
+    </header>
+    <div class="container">
```

Comment:
Why do you wrap the navbar in a <div> when <nav> already shows what it does?

Reply:
My thinking was to have the navbar as part of the container div for the main content of the page but maybe the div might not be needed altogether.

Reply:
Thanks for your review! I find @derfabio's reply to be solid here. Wrapping the non-header content in a container div makes sense.

I'd even go as far as to say this showcases the best use of divs: for division!

What do you think, @Hanmuni ?

Reply:
In my opinion, it's more understandable to relocate the navbar to the header section. When you first open the browser, in my opinion, this structure is better to understand:

   ```
<header>
        <h1>Welcome to the Animal Farm Webshop</h1>
        <nav></nav>
    </header>

    <main>
        <section></section>

        <section></section>

        <aside></aside>
    </main>

    <footer></footer>
```

Having the <main> tag in your browser lets the user know that the main content is in there. A <div class="container"> could mean anything.

Reply:
Ahhh! That makes lots of sense. What do you think, @derfabio ?

Reply:
I think @Hanmuni 's way of structuring the page makes a lot of sense.
It gives a clean view of what is going on in the page, I like it 👍🏻